src/components/NewCommentForm.tsx

```diff
@@ -14,24 +74,35 @@ export const NewCommentForm: React.FC = () => {
             name="name"
             id="comment-author-name"
             placeholder="Name Surname"
-            className="input is-danger"
+            className={`input ${noName && 'is-danger'}`}
+            value={authorName}
+            onChange={(e) => {
+              setNoName(false);
+              setAuthorName(e.target.value);
+            }}
           />
-
           <span className="icon is-small is-left">
             <i className="fas fa-user" />
           </span>
-
-          <span
-            className="icon is-small is-right has-text-danger"
-            data-cy="ErrorIcon"
-          >
-            <i className="fas fa-exclamation-triangle" />
-          </span>
+          {
+            noName
+          && (
```

Comment:
Check the code style everywhere 
```suggestion
          {noName && (
```